installation.md

```diff
@@ -0,0 +1,71 @@
+# Installing Leapp tool
+
+* Prerequisites
+    * Enable copr repository
+        ```shell
+        sudo dnf install dnf-plugins-core -y
```

Comment:
TBH this is actually not the right place to use anymore and inconsistent with moth of the things you explain to install via git below. 

leapp-tool on top of that is quite outdated but the last stable release and we are now trying to get to get leappctl to become the new CLI interface so we would have to focus on that there.

## leapp-tool
I recommend you to follow this for the last stable environment:

https://github.com/leapp-to/leapp/blob/2018-backup/docs/source/getstarted.rst

## leappctl
We should start using https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/ as a source

Reply:
True, I would suggest to have two approaches, one described here will be how to use rpms. The other, on the same page will be a short mention of dev-env where there is thoroughly described how to use automake with our project. This will give the user ways how to use the project on RPM based distros as well as any other, if he/she would like to.